libiocage/lib/Jail.py

```diff
@@ -369,6 +370,56 @@ def _destroy_jail(self):
             stderr=subprocess.DEVNULL
         )
 
+    @property
+    def _dhcp_enabled(self):
+        """
+        True if any ip4_addr uses DHCP
+        """
+        if self.config["ip4_addr"] is None:
+            return False
+
+        return ("dhcp" in self.config["ip4_addr"].networks)
```

Comment:
Redundant parenthesis

Reply:
Hehehe.. But it's a bool :neckbeard: I'm so used to have operators that resolve to a boolean value in parenthesis.